### PR TITLE
Add the Separator Bar and icon_Images fro demo purpose only

### PR DESCRIPTION
**Pull request by instoor**

### CODE REVIEW

File: MCPTT/UI/Channel/MenuBar/ChannelMenuBar.swift [hunk 1]
```diff
@@ -53,6 +55,20 @@ class  ChannelMenuBar: UIView, UICollectionViewDelegate, UICollectionViewDataSou
         horizontalBarView.heightAnchor.constraint(equalToConstant: 6).isActive = true
     }
     
+    func setupSeparatorBar() {
```
Comment (rbniranjanharman) [inline MCPTT/UI/Channel/MenuBar/ChannelMenuBar.swift]: create this method as private

File: MCPTT/UI/Channel/MenuBar/ChannelMenuBar.swift [hunk 2]
```diff
@@ -53,6 +55,20 @@ class  ChannelMenuBar: UIView, UICollectionViewDelegate, UICollectionViewDataSou
...
+        let SeparatorBarView = UIView()
```
Comment (rbniranjanharman) [inline MCPTT/UI/Channel/MenuBar/ChannelMenuBar.swift]: variablename should be start from small letter

File: MCPTT/UI/Channel/ChannelList/channelCell.swift [hunk 3]
```diff
@@ -14,11 +14,13 @@ class channelCell: BaseCell {
     @IBOutlet weak var channelIconImage: UIImageView!
     @IBOutlet weak var channelName: UILabel!
     @IBOutlet weak var channelStatusIcon: UIImageView!
-    @IBOutlet weak var countStatusLabel: UIImageView!
+    @IBOutlet weak var countStatusLabel: UILabel!
     @IBOutlet weak var statusDescriptionIconImage: UIImageView!
     @IBOutlet weak var statusDescriptionLabel: UILabel!
     @IBOutlet weak var separatorView: UILabel!
     
+    @IBOutlet weak var statusLabelLeadingConstraints: NSLayoutConstraint!
```
Comment (rbniranjanharman) [inline MCPTT/UI/Channel/ChannelList/channelCell.swift]: If the variable and method used in local file. Please recheck your all code. Create them as private. channelCell.swift file should be ChannelCell.swift

Comment (instoor) [reply MCPTT/UI/Channel/ChannelList/channelCell.swift]: i have changed the class name and made variable as private where it is required.

**Pull request merged**